### PR TITLE
Change spaces to underscores in filenames, add README comment

### PR DESCRIPTION
**Pull request by riyazdf**
As suggested in #5, uses strings.Replace to change spaces to underscores.

(Recreated, original closed by mistake)

R: @sul3n3t

### CODE REVIEW

File: cmd/request_cert.go [hunk 1]
```diff
@@ -74,7 +74,9 @@ func newCertAction(c *cli.Context) {
 		os.Exit(1)
 	}
 
-	if depot.CheckCertificateSigningRequest(d, name) || depot.CheckPrivateKey(d, name) {
+  formattedName := strings.Replace(name, " ", "_", -1)
```
Comment (sul3n3t) [inline cmd/request_cert.go]: indentation

**Comment by sul3n3t** [comment]
LGTM after indentation fixed

**Comment by riyazdf** [comment]
fixed indentation, I'll update the keywhiz wiki and merge.

**Comment by sul3n3t** [comment]
:+1:

**Pull request merged**